package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@daily-co/react-native-webrtc",
-  "version": "106.0.6-daily.1",
+  "version": "111.0.0-daily.1",
```

Comment:
at some point we switched from being 0-indexed (starting at `-daily.0`) to being 1-indexed?

Reply:
Maybe I did that without intention on the previous release, but I can fix that and return to be `0`.